zoo-css-challenge/style.css

```
@@ -96,6 +113,7 @@ footer {
 
 #badge:hover {
   background-color: yellow;
+  transition: 0.5s;
```

```suggestion
  transition: background-color 0.5s;
```
It's good practise to specify which CSS property we want to animate, for performance reasons

---

Thank you very much. I'll be sure to follow your tips